src/components/shared/MarkerContextMenu.js

```diff
@@ -364,6 +381,20 @@ class MarkerContextMenuImpl extends PureComponent<Props> {
             Copy as JSON
           </Localized>
         </MenuItem>
+        {data && data.type === 'IPC' ? (
+          <MenuItem onClick={this.selectOtherThreadForIPCMarkers}>
+            <span className="react-contextmenu-icon markerContextMenuSelectThread" />
+            {data.direction === 'sending' ? (
```

Comment:
should we check whether `recvTid` is not null? I see that `selectOtherThreadForIPCMarkers` does this check, but it's weird that we would display the context menu item and that it does nothing.

Alternatively it could be `disabled` with a `title` attribute (that you can specify in in the prop `attributes`).